### PR TITLE
[tech] fix double build

### PR DESCRIPTION
**Pull request by PaolaDMadd-Pro**
## What? 
Jira ticket [EPP-117](https://collaboration.homeoffice.gov.uk/jira/browse/EPP-117) it is a tech ticket to fix a double build when we push a change in a branch that has an open PR
## Why? 
To avoid double build when push occurs 
## How? 
update `drone.yaml`
## Testing?
n/a
## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

**Review by Jholderdevops — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -21,7 +21,6 @@ include_default_branch: &include_default_branch
 include_default_and_feature_branches: &include_default_and_feature_branches
   include:
     - master
-    - EPP-*
```
Comment (Jholderdevops) [inline .drone.yml:24]: change this to
`feature/*`

**Review by Jholderdevops — COMMENTED** [review]
see comment

File: .drone.yml (hunk 1 above)
Comment (PaolaDMadd-Pro) [reply .drone.yml:24]: done

Comment (Jholderdevops) [reply .drone.yml:24]: space inbetween please
`- feature/*`

Comment (PaolaDMadd-Pro) [reply .drone.yml:24]: done

**Pull request merged**